contrib/sssd.spec.in

```diff
@@ -54,6 +54,7 @@ Summary: System Security Services Daemon
 License: GPLv3+
 URL: https://github.com/SSSD/sssd/
 Source0: %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+Source1: sssd.sysusers
```

Comment:
This file has to be maintained in downstream dist-git like spec-file?

Reply:
Yes.